src/compiler/compile/render_dom/index.ts

```diff
@@ -226,18 +226,17 @@ export default function dom(
 		component.rewrite_props(({ name, reassigned }) => {
 			const value = `$${name}`;
 
-			const callback = `$value => { ${value} = $$value; $$invalidate('${value}', ${value}) }`;
-
 			if (reassigned) {
-				return `$$subscribe_${name}()`;
+				return b`${`$$subscribe_${name}`}()`;
 			}
 
 			const component_subscribe = component.helper('component_subscribe');
+			const callback = x`$$value => { ${value} = $$value; $$invalidate('${value}', ${value}) }`;
```

Comment:
I think this $$invalidate call should be inlined like so:
```diff
- const callback = x`$$value => { ${value} = $$value; $$invalidate('${value}', ${value}) }`;
+ const callback = x`$$value => $$invalidate('${value}', ${value} = $$value)`;
```

Reply:
you're right, it should — fixing. thanks